content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx

```diff
@@ -73,6 +73,19 @@ Wrangler is the official Cloudflare Worker CLI. You will use it to develop and d
 
 ```terminal
 npx wrangler init
```

Comment:
Maybe mention somehwere that this uses wrangler v2 and link to their docs?
https://developers.cloudflare.com/workers/wrangler/